client/src/assets/scss/main.scss

```diff
@@ -113,21 +112,102 @@ a:active {
   border-style: solid;
   border-radius: 10px;
   background-color: #ffffff;
-  padding-bottom: 5%;
+  padding-bottom: 1%;
   margin-top: 2%;
   margin-left: 10%;
   margin-right: 10%;
 }
 .pasta-title {
   font-family: Georgia, "Times New Roman", Times, serif;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 1%;
+  width: 50%;
+  background-color: #c92c20;
+  text-align: center;
+  color: #fcd32c;
+  border-radius: 10px;
 }
 
 .pasta-desc {
   font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 16pt;
+  padding-top: 3%;
+  padding-left: 3%;
 }
 
 .pasta-category {
   font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  padding-left: 3%;
+}
+
+.profile-page {
```

Comment:
This file is getting MASSIVE with styling, making it difficult to find the style we're looking for if we wanted to change it. Instead, we should start to create separate SCSS files to organize the styles based on page or component. Separate files can be imported into this `main` file since it is already imported into the React app for us and can manage the additional file imports